src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -119,6 +119,15 @@
     text: |
         az keyvault create --location westus2 --name MyKeyVault --resource-group MyResourceGroup
     crafted: true
+  - name: Create a key vault with network ACLs specified (use JSON string).
+    text: |
+        az keyvault create --location westus2 --name MyKeyVault --resource-group MyResourceGroup --network-acls "{\\"ip\\": [\\"1.2.3.4\\", \\"2.3.4.0/24\\"], \\"vnet\\": [\\"vnet_name_1/subnet_name1\\", \\"vnet_name_2/subnet_name2\\", \"/subscriptions/000000-0000-0000/resourceGroups/MyResourceGroup/providers/Microsoft.Network/virtualNetworks/MyVNet/subnets/MySubnet\"]}"
+  - name: Create a key vault with network ACLs specified (use JSON file).
+    text: |
+        az keyvault create --location westus2 --name MyKeyVault --resource-group MyResourceGroup --network-acls network-acls-example.json
+  - name: Create a key vault with network ACLs specified (use --network-acls, --network-acls-ips and --network-acls-vnets at the same time).
+    text: |
+        az keyvault create --location westus2 --name MyKeyVault --resource-group MyResourceGroup --network-acls network-acls-example.json --network-acls-ips 3.4.5.0/24 4.5.6.0/24 --network-acls-vnets vnet1/subnet1 vnet2/subnet2 /subscriptions/000000-0000-0000/resourceGroups/MyResourceGroup/providers/Microsoft.Network/virtualNetworks/vnet3/subnets/subnet3
```

Comment:
thanks @bim-msft for quick change. Should we fail validation when json and flatten params from user at same time?

Reply:
> thanks @bim-msft for quick change. Should we fail validation when json and flatten params from user at same time?

@yungezz I think it's safe to pass validation for this case which all three params are specified at the same time. As far as I can see, there is one scenario may benefits from this: user has one basic network rules json file, and this file is always applied to a new created vault, also there are different additional rules for each new vault.

Reply:
:)